p2p/src/peer_manager/peers_eviction/mod.rs

```diff
@@ -194,16 +186,62 @@ pub fn select_for_eviction_inbound(
     find_group_most_connections(candidates)
 }
 
+/// Outbound block relay connections younger than this age will not be taken into account
+/// during eviction.
+/// Note that extra block relay connections are established and evicted on a regular basis
+/// during normal operation. So, this interval basically determines how often those extra
+/// connections will come and go.
+const BLOCK_RELAY_CONNECTION_MIN_AGE: Duration = Duration::from_secs(120);
+
+/// Outbound full relay connections younger than this age will not be taken into account
+/// during eviction.
+/// Note that extra full relay connections are established if the current tip becomes stale.
+/// There is no point in making this interval large, it should just give the peer enough
+/// time to send us a block.
+const FULL_RELAY_CONNECTION_MIN_AGE: Duration = Duration::from_secs(30);
```

Comment:
Isn't 30 seconds too short for this?

Reply:
I'm not sure. In bitcoin they use 30 seconds too (MINIMUM_CONNECT_TIME), this made me think it should be enough. (But note that they use the same 30s for extra block relay connections as well)

Reply:
I made it 120 sec, the same as for block relay connections.